app/circleSettings.js

```diff
@@ -0,0 +1,148 @@
+import { Member } from '../components/Member.js';
+
+const StyledSafeArea = styled(SafeAreaView);
+const StyledView = styled(View);
+const StyledText = styled(Text);
+const StyledAnimatedView = styled(Animated.createAnimatedComponent(View));
+const StyledScrollView = styled(ScrollView);
+
+export default function Page() {
+
+	const [modalVisible, setModalVisible] = useState(false);
+
+	const [isEnabled, setIsEnabled] = useState(false);
+	const toggleSwitch = () => setIsEnabled(previousState => !previousState);
+	const togglePosition = React.useRef(new Animated.Value(1)).current;
+
+	const Trevor = "https://media.licdn.com/dms/image/C4E03AQEjKbD7qFuQJQ/profile-displayphoto-shrink_200_200/0/1574282480254?e=1701907200&v=beta&t=1BizKLULm5emiKX3xlsRq7twzFTqynOsfTlbRwqNuXI";
+
+	React.useEffect(() => {
+		Animated.timing(togglePosition, {
+		toValue: isEnabled ? 45 : 5,
+		duration: 200,
+		useNativeDriver: false,
+		}).start();
+	}, [isEnabled]);
+
+	return (
+		<StyledSafeArea className='bg-offblack border' style={{ flex: 1 }}>
+			<StyledView className='flex-1 items-center'>
+				<Button
+					btnStyles='sticky absolute left-5 top-10 bg-grey rotate-180'
+					height={'h-[60px]'}
+					width={'w-[60px]'}
+					iconSize={40}
+					icon='log-out'
+					iconColor='#F9A826'
+        			onPress={() => setModalVisible(true)}
+				/>
+				<Button
+					btnStyles='sticky absolute right-5 top-10 bg-grey'
+					height={'h-[60px]'}
+					width={'w-[60px]'}
+					iconSize={40}
+					icon='trash'
+					iconColor='#CC2500'
+				/>
+				<StyledText className='absolute top-10 text-3xl text-offwhite bg-grey text-center h-[60px] px-[35px] py-[12px] rounded-full'>
+					Settings
+				</StyledText>
+
+				{/* DOES NOT WORK! */}
+
+				<Modal
+					animationType="slide"
+					transparent={false}
+					visible={modalVisible}
+					onRequestClose={() => {
+					Alert.alert('Modal has been closed.');
+					setModalVisible(!modalVisible);
+					}}>
+					<StyledView>
+						<StyledView>
+							<StyledText>Hello World!</StyledText>
+							<StyledText onPress={() => setModalVisible(!modalVisible)}>Hide Modal</StyledText>
+						</StyledView>
+					</StyledView>
+				</Modal>
+
+				<StyledView className='top-[130px] w-[85%] gap-y-8 flex'>
+					<StyledView className="bg-grey h-[60px] py-[9px] px-[50px] rounded-xl items-center flex flex-row">
+						<Button
+							btnStyles='inline bg-grey border-2 border-purple mr-3'
+							height={'h-[45px]'}
+							width={'w-[45px]'}
+							iconSize={30}
+							icon='musical-notes'
+							iconColor='white'
+						/>
+						<StyledText className='inline font-bold text-3xl text-offwhite'>
+							Circle Name
+						</StyledText>
+					</StyledView>
+					<StyledView className="bg-grey h-[60px] py-[12px] rounded-xl pl-5 flex flex-row">
+						<StyledText className='font-bold text-3xl text-offwhite'>
+							Notifications
+						</StyledText>
+						<TouchableOpacity onPress={toggleSwitch}>
```

Comment:
I think it's cool that you made this but maybe we should just use the built in React Native switch? I think we should have a good bit of customization with that and it has state management which this is missing.